addons/mrp/models/mrp_production.py

```diff
@@ -208,8 +208,7 @@ def _compute_unreserve_visible(self):
     @api.depends('move_raw_ids.quantity_done', 'move_finished_ids.quantity_done')
     def _compute_post_visible(self):
         for order in self:
-            order.post_visible = any(order.move_raw_ids.filtered(lambda x: (x.quantity_done) > 0 and (x.state not in ['done', 'cancel']))) or \
-                any(order.move_finished_ids.filtered(lambda x: (x.quantity_done) > 0 and (x.state not in ['done', 'cancel'])))
+            order.post_visible = any(order.move_finished_ids.filtered(lambda x: (x.quantity_done) > 0 and (x.state not in ['done', 'cancel'])))
```

Comment:
The lambda should not even be necessary, you can do the whole thing as a generator comprehension inside the `any` call.